opensrp-chw-referral/src/main/java/org/smartregister/chw/referral/activity/BaseReferralHistoryActivity.java

```diff
@@ -0,0 +1,79 @@
+package org.smartregister.chw.referral.activity;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+import android.widget.TextView;
+
+import org.joda.time.DateTime;
+import org.joda.time.Period;
+import org.smartregister.chw.referral.R;
+import org.smartregister.chw.referral.contract.BaseReferralHistoryContract;
+import org.smartregister.chw.referral.domain.MemberObject;
+import org.smartregister.chw.referral.presenter.BaseferralHistoryPresenter;
+import org.smartregister.chw.referral.util.Constants;
+
+import java.util.Collection;
+import java.util.Locale;
+
+public class BaseReferralHistoryActivity extends AppCompatActivity implements BaseReferralHistoryContract.View {
+    protected MemberObject MEMBER_OBJECT;
```

Comment:
Use camel case convention in naming of variable since this is not a constant field

Reply:
Noted